src/utils/components/PreferenceHelpTextIcon/PreferenceHelpTextIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
```

Comment:
import React from 'react' can be used

Reply:
Of course, it can. Just IMO it didn't worth to do that for such a little component.. We could find many places across our codebase where we could do that.. and I wonder why we didn't do that from the beginning everywhere and by default..

Reply:
we didn't do it from the beginning because the tsconfig file wasn't updated so we weren't able, since we update the tsconfig file and we can do it, we try to keep the import in this preferred way :)